cmd/jimmctl/cmd/role.go

```diff
@@ -0,0 +1,384 @@
+// Copyright 2024 Canonical.
+
+package cmd
+
+import (
+	"bufio"
+	"fmt"
+	"strings"
+
+	"github.com/juju/cmd/v3"
+	jujucmdv3 "github.com/juju/cmd/v3"
+	"github.com/juju/gnuflag"
+	jujuapi "github.com/juju/juju/api"
+	jujucmd "github.com/juju/juju/cmd"
+	"github.com/juju/juju/cmd/modelcmd"
+	"github.com/juju/juju/jujuclient"
+
+	"github.com/canonical/jimm/v3/internal/errors"
+	"github.com/canonical/jimm/v3/pkg/api"
+	apiparams "github.com/canonical/jimm/v3/pkg/api/params"
+)
+
+var (
+	roleDoc = `
+role command enables role management for jimm
+`
+
+	addRoleDoc = `
+add command adds role to jimm.
+
+Example:
+	jimmctl auth role add <name> 
+`
+	renameRoleDoc = `
+rename command renames a role in jimm.
+
+Example:
+	jimmctl auth role rename <name> <new name>
+`
+	removeRoleDoc = `
+rename command removes a role in jimm.
+
+Usage:
+-y	Remove role without promping for confirmation
```

Comment:
Broken formatting?

Reply:
Appears not, checked group its the same. Also noticed we don't have "usage" in other commands... I'll leave as is for when we revisit jimmctl